code/game/machinery/autolathe.dm

```diff
@@ -1,6 +1,7 @@
 /obj/machinery/autolathe
 	name = "autolathe"
 	desc = "It produces items using iron, glass, plastic and maybe some more."
+	icon = 'icons/obj/machines/light_lathes.dmi'
```

Comment:
What makes it light?

Reply:
comparasment to heavy_lathe

Reply:
Oh weird, I hadn't seen the heavy lathe file before! That's just the [E.X.P.E.R.I-MENTOR](https://wiki.beestation13.com/view/E.X.P.E.R.I-MENTOR), not technically a lathe. Might be better to rename light_lathes to lathes and either include the experimentor in it or rename heavy_lathe to something else